### PR TITLE
Fix node version incompatibility

### PR DESCRIPTION
**Pull request by horia-delicoti**
The result of command in node v8 of: _process.binding('constants').fs.hasOwnProperty_ is _undefined_
---- Autogenerated Waffleboard Connection: Connects to #42

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - "6"
```
Comment (jviotti) [inline .travis.yml]: Can we add `8` but keep `6`, so we test in both versions?

File: .travis.yml [hunk 2]
```diff
@@ -1,6 +1,7 @@
 language: node_js
 node_js:
   - "6"
+  - "8"
```
Comment (jhermsmeier) [inline .travis.yml]: Should Node 4 also be added here? There are still a lot of people on 4, and most of our other modules maintain compatibility with it.

Comment (jhermsmeier) [reply .travis.yml]: ping @horia-delicoti ^

Comment (zvin) [reply .travis.yml]: The tests were failing on node 4: https://travis-ci.org/resin-io/node-lkl/jobs/290434898
I needed this merged for node 8 support.
We can add node 4 support in a new PR.

**Pull request merged**